Symfony/CS/Tests/Fixer/PSR2/ClassDefinitionFixerTest.php

```diff
@@ -154,6 +154,14 @@ public function provideAnonymousClassesCases()
                 "<?php \$a = new class(10) extends SomeClass implements SomeInterface, D\n{};",
                 "<?php \$a = new    class(10)     extends\nSomeClass\timplements    SomeInterface, D {};",
             ),
+            array(
+                "<?php \$a = new class(\$this->prop)\n{};",
+                '<?php $a = new class(   $this->prop   ){};',
```

Comment:
I agree. It would follow PSR-2 anonymous functions standards + PSR-12 anonymous classes standards